<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="FCU-NBK500" failures="4" tests="4" name="Test-class test" time="127.226" errors="0" timestamp="2020-07-29T17:49:10 MESZ">
  <testcase classname="de.alain.TestNGParallel.TestParallelClassTwo" name="test_2" time="30.942">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="Index 1 out of bounds for length 1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
at de.alain.ProjektStatitikUndInformationen.ProjektStatistik.GesamteProjekt(ProjektStatistik.java:27)
at de.alain.ProjektmaskeNavigationUndProjektmerkmaleHolen.ProjektMaskeNavigationUndProjektMerkmaleHolen.ProjektmaskeAnklickeUndMaskeInformationenAssortieren(ProjektMaskeNavigationUndProjektMerkmaleHolen.java:38)
at de.alain.TestNGParallel.TestParallelClassTwo.test_2(TestParallelClassTwo.java:114)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- test_2 -->
  <testcase classname="de.alain.TestNGParallel.TestParallelClassTwo" name="test_2" time="23.032">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="Index 1 out of bounds for length 1">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
at de.alain.ProjektStatitikUndInformationen.ProjektStatistik.GesamteProjekt(ProjektStatistik.java:27)
at de.alain.ProjektmaskeNavigationUndProjektmerkmaleHolen.ProjektMaskeNavigationUndProjektMerkmaleHolen.ProjektmaskeAnklickeUndMaskeInformationenAssortieren(ProjektMaskeNavigationUndProjektMerkmaleHolen.java:38)
at de.alain.TestNGParallel.TestParallelClassTwo.test_2(TestParallelClassTwo.java:114)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- test_2 -->
  <testcase classname="de.alain.TestNGParallel.TestParallelClassOne" name="test_1" time="107.822">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.fail(Assert.java:95)
at de.alain.PageAndHTMLControl.Base.elementsIsVisible(Base.java:175)
at de.alain.PageAndHTMLControl.Login.isEmailVisible(Login.java:158)
at de.alain.TestNGParallel.TestParallelClassOne.test_1(TestParallelClassOne.java:105)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- test_1 -->
  <testcase classname="de.alain.TestNGParallel.TestParallelClassOne" name="test_1" time="10.531">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.fail(Assert.java:95)
at de.alain.PageAndHTMLControl.Base.elementsIsVisible(Base.java:175)
at de.alain.PageAndHTMLControl.Login.isEmailVisible(Login.java:158)
at de.alain.TestNGParallel.TestParallelClassOne.test_1(TestParallelClassOne.java:105)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- test_1 -->
</testsuite> <!-- Test-class test -->
